src/nspawn/nspawn.c

```diff
@@ -676,9 +676,8 @@ static int parse_argv(int argc, char *argv[]) {
                                 r = free_and_strdup(&arg_machine, optarg);
                                 if (r < 0)
                                         return log_oom();
-
-                                break;
                         }
+                        break;
```

Comment:
Indeed, this fixes `-Z some-context -M ""`. But why do we allow an empty `-M`?

I'm reading eb91eb187b7491e05, but I don't understand